jupyterlab_search_replace/handlers.py

```diff
@@ -1,6 +1,7 @@
 import json
 
 import tornado
+import asyncio
```

Comment:
Small ordering of the import.

```suggestion
import asyncio
import json

import tornado
```